acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -135,6 +135,7 @@ const langFromPath = url.pathname.split('/')[1];
 const pageLang = localeMap[langFromPath] || 'en-us';
 
 export default async function init(element) {
+  if (document.querySelector('div[data-section="widget"]')) return;
```

Comment:
duplication was happening at the edge worker. Not sure this will help, but probably can't hurt either?

Reply:
Do you mean what we saw was two `<section id="edge-snippet">`s? Anyway, the change is to safeguard `init()` so it won't be called twice or more.

Reply:
Yes, I believe so.